lib/ansible/executor/module_common.py

```diff
@@ -1148,6 +1148,8 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
                 display.debug('ANSIBALLZ: Lock acquired: %s' % id(lock))
                 # Check that no other process has created this while we were
                 # waiting for the lock
+                #moved from 1246
+                now = datetime.datetime.utcnow()
```

Comment:
I may not have been very clear in my alternate suggestion, but this is not the only change required. Effectively we'd have to create and write a `ZipInfo` object that uses the datetime object to pass in a timetuple based off of `now`. And then use `writestr` to write the `ZipInfo` into the zipfile, which would be a much larger change than my first suggestion of just checking the year.